code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1230,7 +1230,7 @@
 	result_amount = 3
 
 /datum/chemical_reaction/drinks/nuka_cola
-	name = "Nuka Cola"
+	name = "Nuclear Cola"

```

Comment:
You can, I just wanted to get rid of the 'lawsuit pls' name and won't be touching it further.